test/unit/client/test_client.py

```diff
@@ -387,6 +424,31 @@ def test_create_enterprise_user_returns_the_correct_user_object(mock_client, moc
     assert new_user.name == test_user_name
 
 
+def test_terms_of_service_returns_the_correct_tos_object(mock_client, mock_box_session, terms_of_service_response):
+    # pylint:disable=redefined-outer-name
+    value = json.dumps({
+        'status': 'enabled',
+        'tos_type': 'external',
+        'text': 'This is a test text',
+    })
+    mock_box_session.post.return_value = terms_of_service_response
+    new_terms_of_service = mock_client.create_a_terms_of_service('enabled', 'external', 'This is a test text')
+    assert len(mock_box_session.post.call_args_list) == 1
+    assert mock_box_session.post.call_args[0] == ("{0}/terms_of_services".format(API.BASE_API_URL),)
+    assert mock_box_session.post.call_args[1] == {'data': value}
```

Comment:
This can probably all be simplified down to `mock_box_session.post.assert_called_once_with()`